docs/group_guidelines/git_standards.md

```diff
@@ -0,0 +1,86 @@
+# Git - Standard Practices
+The purpose of this document is to provide our team with a set of standards to follow throughout our capstone project.
+With this document, we hope to ensure that we stay organized and consistent throughout the term.
+
+# Branches
+
+We will be following a technique called feature branches, where we create a new branch for every feature which is to be
+developed. This allows us to prevent merge conflicts, and closely monitor each new feature we need to create. As well,
+feature branches ensure we are dividing properly dividing issues into achievable tasks. Once the feature has been 
+completed, and the branch has been merged to main, we will NOT be deleting the branch. If bugs occur with the feature, 
+we can return to the branch in order to fix the bug. 
```

Comment:
I like that we'll be using the branches for bugs, too. Makes it easy to see the history of features (whether it's working, buggy, getting fixed, etc.(

Reply:
Yes I agree, a  "hot fix" branch is what they are usually called!